src/services/user/hooks/userService.js

```diff
@@ -414,28 +416,15 @@ const enforceRoleHierarchyOnDelete = async (context) => {
 	return enforceRoleHierarchyOnDeleteBulk(context);
 };
 
-const generateRegistrationLink = async (context) => {
-	const { data, app } = context;
-	if (data.generateRegistrationLink === true) {
-		delete data.generateRegistrationLink;
-		if (!data.roles || data.roles.length > 1) {
-			throw new BadRequest('Roles must be exactly of length one if generateRegistrationLink=true is set.');
-		}
-		const { hash } = await app.service('/registrationlink')
-			// set account in params to context.parmas.account to reference the current user
-			.create({
-				role: data.roles[0],
-				save: true,
-				patchUser: true,
-				host: SC_DOMAIN,
-				schoolId: data.schoolId,
-				toHash: data.email,
-			})
-			.catch((err) => {
-				throw new GeneralError(`Can not create registrationlink. ${err}`);
-			});
-		context.data.importHash = hash;
+
+const sendRegistraionLink = async (context) => {
```

Comment:
```suggestion
const sendRegistrationLink = async (context) => {
```

needs to be updated in other files too